client/src/components/dashboard/Graph/PieChart.js

```diff
@@ -0,0 +1,156 @@
+import HighchartsReact from 'highcharts-react-official';
+import Highcharts from 'highcharts/highstock';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * This is the Pie Chart component
+ * This has two variants
+ * 1. This plots the no of clicks made in the last month per location on a pieChart
+ * 2. This plots the no of clicks made in the last month per url on a pieChart
+ * @usage Just import it and pass a variant prop
+ * whose value should be 'Clicks_by_location' for 1 else Clicks_by_urls for 2
```

Comment:
```suggestion
 * whose value should be `CLICKS_BY_LOCATION` for 1 else `CLICKS_BY_URL` for 2
```
+1 for the point that you are re-using components.
Try using constant like string, makes more intuitive imo.